SeedTotem/SeedTotemMod.cs

```diff
@@ -102,14 +96,23 @@ public void OnDestroy()
             harmony?.UnpatchAll(PluginGUID); 
         }
 
-        private void AddCustomPrefabs(On.ObjectDB.orig_CopyOtherDB orig, ObjectDB self, ObjectDB other)
+        private void AddCustomPrefabs()
         {
+            try { 
+            //Gotta fix this up to work with new Jotunn too (well idk how the old way was TBH)
             seedTotemPrefabConfig = new SeedTotemPrefabConfig();
 
-            GameObject seedTotemPrefab = PrefabManager.Instance.CreateClonedPrefab(SeedTotemPrefabConfig.prefabName, "guard_stone");
+            var seedTotemPrefab = PrefabManager.Instance.CreateClonedPrefab(SeedTotemPrefabConfig.prefabName, "guard_stone");
             seedTotemPrefabConfig.UpdateCopiedPrefab(seedTotemPrefab);
-
-            orig(self, other);
+            }
+            catch (Exception ex)
+            {
+                Jotunn.Logger.LogError($"Error while adding cloned item: {ex.Message}");
+            }
+            finally
+            {
+                ItemManager.OnVanillaItemsAvailable -= AddCustomPrefabs;
```

Comment:
@sbtoonz Is this required/best practice? I haven't seen any other mods do this, but it does make sense, most seem to use a boolean to avoid doing this twice. This avoids an otherwise useless field though, so you may be on to something here 👍

Reply:
I would consider it a "better" practice IMHO I was following @sirskunksalot lead from the examplemod.cs as the "golden" rule if you will 

seemed logical enough for an approach with desired results

Reply:
Also wanna point out that I am NO professional C# coder I am a linux guy and a C monkey ;) hahaha